vagrant/scripts/install_iml_local.sh

```diff
@@ -119,8 +134,8 @@ rm -rf /tmp/iml/_topdir/
 su -l mocker << EOF
 mock -r /etc/mock/iml.cfg --init
 mock -r /etc/mock/iml.cfg --copyin /integrated-manager-for-lustre /iml
-mock -r /etc/mock/iml.cfg -i cargo git ed epel-release python-setuptools gcc openssl-devel postgresql96-devel python2-devel python2-setuptools ed
-mock -r /etc/mock/iml.cfg --shell 'export CARGO_HOME=/tmp/.cargo CARGO_TARGET_DIR=/tmp/target && cd /iml && make local'
+mock -r /etc/mock/iml.cfg -i cargo git ed epel-release python-setuptools gcc openssl-devel postgresql96-devel python2-devel python2-setuptools ed yum
```

Comment:
We should not need yum

Reply:
It's not available in the mock environment but it is used in the srpm task.

Reply:
Where is it being used?

Reply:
https://github.com/whamcloud/integrated-manager-for-lustre/blob/83d065653a55ee4c1a6068c1d024f02a02c73172/.copr/Makefile#L126

Reply:
That should be removed from the srpm and yarn should be installed here.

Reply:
I thought about doing that but was hesitant because I thought it might break it somewhere else, like in CI. I guess I would just need to update it for the docker builder as well right?

Reply:
Most likely. But we should be avoiding doing yum / dnf etc... calls inside the makefile.

Reply:
Ok, I will move it out of the makefile and into here and update the docker builder as well.

Reply:
I've pulled this out to a separate patch.